### PR TITLE
Update ServiceNow.com.xml

### PR DESCRIPTION
**Pull request by gopherit**
Adding docs subdomain.  Removing knowledge -> community rewrite rule.

### CODE REVIEW

File: src/chrome/content/rules/ServiceNow.com.xml [hunk 1]
```diff
@@ -55,19 +54,16 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="servicenow.com" />
+	<target host="www.servicenow.com" />
 	<target host="community.servicenow.com" />
 	<target host="developer.servicenow.com" />
+	<target host="docs.servicenow.com" />
 	<target host="express.servicenow.com" />
+	<target host="knowledge.servicenow.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/ServiceNow.com.xml]: This host has not the same content on https as on http.

Comment (gopherit) [inline src/chrome/content/rules/ServiceNow.com.xml]: @J0WI can you please provide examples of https and http that aren't the same?  I cannot find any.  I removed the rule from knowledge to community, but http://knowledge is redirected to https://knowledge by their server.

Comment (J0WI) [inline src/chrome/content/rules/ServiceNow.com.xml]: Ah sorry, HTTPS Everywhere currently is rewriting to a different location, but you just fixed this.

Comment (gopherit) [inline src/chrome/content/rules/ServiceNow.com.xml]: Great.  Yeah, it's been a bug for a long time.

**Pull request merged**